packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.controller.js

```diff
@@ -1,9 +1,20 @@
 import { MOBILE_OS } from './softphone.constants';
 
 export default class SoftphoneController {
+  /* @ngInject */
+  constructor($state) {
+    this.$state = $state;
+  }
```

Comment:
Should be defined in the routing file rather
```suggestion
  constructor($state) {
    this.$state = $state;
  }
```